RELEASE-NOTES.md

```diff
@@ -19,6 +19,21 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 
 ## [Unreleased] - Release date yyyy-mm-dd
 
+### Added
+
+### Changed
+
+### Removed
+
+
+## [Version 0.10.0] - Release date 2024-09-xx
+
+### Changed
+- Updates to [Conduit version 0.9.2][https://github.com/LLNL/conduit/releases/tag/v0.9.2]
+- Updates to [RAJA version 2024.07.0][https://github.com/LLNL/RAJA/releases/tag/v2024.07.0]
+- Updates to [camp version 2024.07.0][https://github.com/LLNL/camp/releases/tag/v2024.07.0]
+- Updates to [Umpire version 2024.07.0][https://github.com/LLNL/Umpire/releases/tag/v2024.07.0]
```

Comment:
Please merge these four lines w/ the existing `Changed` section for axom@0.10 a few lines down (below `Added`)

Reply:
I don't understand. This follows what is done for other releases that have updated TPL versions.

Reply:
There are two `Changed` sections for this release -- https://github.com/LLNL/axom/blob/b1f5bbc7c617faaf28643f0f88184ccc7d52eae2/RELEASE-NOTES.md

I can take care of the fix

Reply:
I see what you mean now. I thought the TPL version updates should be up front so they are more obvious.  But, whatever you prefer is fine with me.